chore: add Jenkins documentation to Devops

goobric opened the pull request:
`Closes #1931`

## Fixes Issue

<!-- Remove this section if not applicable -->

Closes #1931

## Changes proposed

Added Jenkins documentation to the Devops category

## Screenshots

<!-- Add all the screenshots which support your changes -->

## Note to reviewers

<!-- Add notes to reviewers if applicable -->
![231012_linkhub_jenkins1](https://github.com/rupali-codes/LinksHub/assets/23141638/0fad6ef7-7bab-4032-92f1-7c58c0d65030)
![231012_linkhub_jenkins2](https://github.com/rupali-codes/LinksHub/assets/23141638/1d03919c-6f9b-43bc-8fc7-6353b9056516)
![231012_linkhub_jenkins3](https://github.com/rupali-codes/LinksHub/assets/23141638/9f3c5e96-068b-4678-8543-f7916ad934eb)

CBID2 commented on database/devops/jenkins.json:
```diff
@@ -0,0 +1,9 @@
+[
+    {
+        "name": "Learn how to automate and deploy software by building projects",
+        "description": "The leading open source automation server, Jenkins provides hundreds of plugins to support building, deploying and automating any project.",
```
```suggestion
"description": "Jenkins provides hundreds of plugins to support building, deploying, and automating any project.",
```

CBID2 reviewed (changes requested):
Well done @goobric. Just needs improvement

Anmol-Baranwal reviewed (changes requested):
@goobric 
You need to add it here: `database\data.ts`

![image](https://github.com/rupali-codes/LinksHub/assets/74038190/c6553722-f0bf-4df9-91db-c8aafec91ecc)


As you can see it's not visible.

![image](https://github.com/rupali-codes/LinksHub/assets/74038190/658f672a-2dce-4f9d-82d4-6e6be785aaba)

Anmol-Baranwal commented:
@goobric 
Why did you delete the head repo?

goobric commented:
hi @Anmol-Baranwal 
I responded to this message yesterday at about 14:10 UTC, using the github app on my mobile/cell phone; yet my response isn't listed!
I updated the data.ts  file with the 'jenkins' information:
- saved the file locally on my cloned repo
- added the file
- committed the file
- pushed the file
received an error message about 'fast-forward' use 'pull' command from forked repo
tried to 'pull' my forked repo; received another error!
So I thought, "maybe it's better to start again" (clearly this was the wrong approach); so I deleted my locally cloned repo; deleted the forked repo from my github account.
- I forked the original repo to my gh account;
- cloned the repo to my local computer
- created a new branch
- added details about 'jenkins' to index, data.ts, CategoryDescription.ts, added jenkins.json in the devops directory
- saved the files locally
- added the files to the stage
- commited the files 
- pushed the files to my forked repo
Everything pushed successfully to my forked repo
- yet, on my repo; there was no option to "Compare & PR" !
What did I do wrong? please explain to me what I should have done, to have a successful PR?

goobric commented:
@Anmol-Baranwal what do I do now?
Do I start the process again from the beginning? I am confused!

Anmol-Baranwal commented:
@goobric Comment on this PR: #1976

Anmol-Baranwal commented:
I mistakenly created a PR, so the PR will be merged as a co-authored commit.

If you want to only do it urself, then you can create a fresh PR.

For now: it will be similar to this, we both will be assigned to the PR.

![image](https://github.com/rupali-codes/LinksHub/assets/74038190/8b5b741d-cac2-41ea-adee-c901c3ee8277)

goobric commented:
@Anmol-Baranwal that's great :-) 
thanks for co-authoring, I have never co-authored a PR before!
This should be a common option on github (probably is already) I would like to co-author on more PRs; but I don't know how.
I am a big advocate of pair-programming and problem solving.
Thanks for creating the PR and merging it.
Have a wonderful weekend.

Anmol-Baranwal commented:
It hasn't been merged, you need to comment there in order for me to assign the PR to you.

Then we can go forward.

goobric commented:
@Anmol-Baranwal 
I am so sorry to ask such a stupid question, but what do I need to say in the comment for the PR to be assigned?
Do I just say, "Please assign this PR me"

![231014_LinksHub_PR1](https://github.com/rupali-codes/LinksHub/assets/23141638/12e0d9e3-3960-42f0-b13e-0621d8a3b6d4)
![231014_LinksHub_PR2](https://github.com/rupali-codes/LinksHub/assets/23141638/7085d32b-1301-4190-aeb9-239ecf93424a)
![231014_LinksHub_PR3](https://github.com/rupali-codes/LinksHub/assets/23141638/e8295ed3-7131-4298-afdd-d7f3b893ce17)

Anmol-Baranwal commented:
@goobric 
Comment anything. GitHub works in a way, that those who have raised or commented can be assigned.

So, it is now showing your option.

As you can see, your name is not available for assignee.

![image](https://github.com/rupali-codes/LinksHub/assets/74038190/2cd1a048-a9a8-466f-8633-2a154409d4eb)

Anmol-Baranwal commented:
See once someone comments.

![image](https://github.com/rupali-codes/LinksHub/assets/74038190/2f134d67-98db-493b-961b-456ea5e19808)

His name is available.
![image](https://github.com/rupali-codes/LinksHub/assets/74038190/d46c0352-6770-46da-8f52-5c16e8481c57)
